dependencies/llvm/CMakeLists.txt

```diff
@@ -75,6 +75,13 @@ endforeach ()
 set(wasm_libs "")
 if (TARGET_WEBASSEMBLY)
     find_package(LLD CONFIG REQUIRED HINTS "${LLVM_DIR}/../lld")
+
+    if (LLVM_LINK_LLVM_DYLIB AND NOT Halide_SHARED_LLVM)
+        message(FATAL_ERROR "LLVM was built with LLVM_LINK_LLVM_DYLIB. "
```

Comment:
It might be useful to reference this PR (and/or issue) in a comment here.

Reply:
Will do

Reply:
Done; merging.